lms/djangoapps/verify_student/views.py

```diff
@@ -856,6 +858,96 @@ def submit_photos_for_verification(request):
     return HttpResponse(200)
 
 
+def _compose_message_reverification_email(
+        course_key, user_id, relates_assessment, photo_verification, status, is_secure
+):  # pylint: disable=invalid-name
+    """ Composes subject and message for email
+
+    Args:
+        course_key(CourseKey): CourseKey object
+        user_id(str): User Id
+        relates_assessment(str): related assessment name
+        photo_verification(QuerySet/SoftwareSecure): A query set of SoftwareSecure objects or SoftwareSecure objec
+        status(str): approval status
+        is_secure(Bool): Is running on secure protocol or not
+
+    Returns:
+        None if any error occurred else Tuple of subject and message strings
+    """
+    try:
+        location_id = VerificationStatus.get_location_id(photo_verification)
+        usage_key = UsageKey.from_string(location_id)
+        course = modulestore().get_course(course_key)
+        redirect_url = get_redirect_url(course_key, usage_key.replace(course_key=course_key))
+
+        subject = render_to_string('emails/reverification_processed_subject.txt', {})
+        # Email subject must not contain newlines
+        subject = ''.join(subject.splitlines())
+
+        context = {
+            "status": status,
+            "course_name": course.display_name_with_default,
+            "assessment": relates_assessment,
+            "courseware_url": redirect_url
+        }
+
+        ver_block = modulestore().get_item(usage_key)
+        allowed_attempts = 1 if ver_block.attempts == 0 else ver_block.attempts
```

Comment:
There is method `remaining_attempts` in `Reverification` block which can be used here to get remaining attempts:
```python
reverification_block = modulestore().get_item(usage_key)
user_attempts = VerificationStatus.get_user_attempts(user_id, course_key, relates_assessment, location_id)
left_attempts = reverification_block.remaining_attempts(user_attempts)
verification_open = True
if left_attempts < 1 or reverification_block.due < timezone.now():
    verification_open = False
```